File: POO/Constructores_Clases/Class/JAVA/CuentaBancaria.java
public class CuentaBancaria {
    private String numeroCuenta;
    private double saldo;
    private boolean activa;

    // Constructor sin parámetros
    public CuentaBancaria() {
        this.numeroCuenta = "000000";
        this.saldo = 0.0;
        this.activa = false;
    }

    public void mostrar() {
        System.out.printf("Cuenta: %s, Saldo: $%.2f, Activa: %b%n", 
            numeroCuenta, saldo, activa);
    }

    public static void main(String[] args) {
        CuentaBancaria cuenta = new CuentaBancaria();
        cuenta.mostrar();
    }
}

File: POO/Constructores_Clases/Class/js/CuentaBancaria.js
class CuentaBancaria {
    constructor() { //Constructor sin parámetros
        this.numeroCuenta = "000000";
        this.saldo = 0.0;
        this.activa = false;
    }

    mostrar() {
        console.log(`Cuenta: ${this.numeroCuenta}, Saldo: $${this.saldo.toFixed(2)}, Activa: ${this.activa}`);
    }
}

// Uso
const cuenta = new CuentaBancaria();
cuenta.mostrar();